README.md

```diff
@@ -29,164 +29,18 @@ Limitations: DuckDB is not designed for frequent CRUD (create, read, update, and
-   pip install -r requirements.txt
-   ```
-
-3. **Configure dbt profiles:**
-
-   dbt uses a `profiles.yml` file to connect to DuckDB. Ensure that your `profiles.yml` file is correctly configured as follows:
-
-   ```yaml
-   duckdb_project:
-     target: dev
-     outputs:
-       dev:
-         type: duckdb
-         path: "path/to/your/database.db"  # Path to your DuckDB database file
-         extensions: ['parquet', 'json']   # Optional extensions to load, if needed
-   ```
-
-4. **Set up the directory structure:**
-
-   Your dbt project should follow this structure:
-
-   ```
-   .
-   ├── models
-   │   ├── staging
-   │   │   └── csv_imports.sql  # SQL to define CSV imports
-   │   ├── transforms.sql       # SQL transformation models
-   |   ├── source.yml  
-   ├── data
-   │   └── source.csv           # Place your CSV files here
-   ├── dbt_project.yml          # dbt project configuration
-   └── README.md                # This file
-   ```
-
----
-
-
-## Usage
-
-### Loading CSV Files
-
-1. **Prepare your CSV files:**
-   - Place the source CSV files inside the `data/` directory.
-
-2. **Create staging models:**
-   - In the `models/staging` folder, create a model (SQL file) that defines how each CSV should be loaded into DuckDB.
-
-   Example SQL to load a CSV:
-
-   ```sql
-   {{ config(materialized='table') }}
-
-   select * 
-   from read_csv_auto('data/source.csv');
-   ```
-
-3. **Run dbt to load and transform data:**
-
-   Once your staging models are ready, run dbt to load the CSVs into DuckDB:
-
-   ```bash
-   dbt run
-   ```
-
-4. **Query the data:**
-
-   After running dbt, you can query your data directly using DuckDB or through dbt models.
-
-   Example:
-
-   ```sql
-   select * from staging.source_csv;
-   ```
-
----
-
-## Example
-
-Here is an example walkthrough:
-
-1. Place a CSV file, `sales_data.csv`, in the `data/` folder.
-2. Create a corresponding SQL file, `sales_data.sql`, in the `models/staging/` folder:
-
-   ```sql
-   {{ config(materialized='table') }}
-
-   select * 
-   from read_csv_auto('data/sales_data.csv');
-   ```
-
-3. Run dbt to load the data:
-
-   ```bash
-   dbt run
-   ```
-
-4. Query the loaded `sales_data` table:
-
-   ```sql
-   select * from staging.sales_data;
-   ```
+Now most of data engineering work is moved to `sei_dbt` folder. The instructions are available in `readme.md` in `sei_dbt` folder. 
 
 ---
 
 ## Project Status
 
-This project is currently in active development. The core functionality of loading CSV files into DuckDB using dbt is complete. Future improvements may include:
+This project is currently in active development. 
+The core functionality of loading CSV files into DuckDB using dbt is complete. 
 
-- Support for additional file formats (e.g., Parquet, JSON).
-- Enhanced data validation and error handling.
-- Automating more transformations and data processing workflows.
+Some tests in terms of the reading speed are done in `05_comparison_dplyr_duckdb.r`.
```

Comment:
This line should be up in the overview section.

Reply:
Agree. And moved it to overview section.